test/odbc/mssqlodbc/test_data_types.cpp

```diff
@@ -448,3 +448,93 @@ TEST_F(MSSQL_Data_Types, Varchar) {
 
   DataTypesTestCommon(TEST_TABLE, TABLE_COLUMNS, INSERTED_VALUES, EXPECTED_VALUES);
 }
+
+// Test retrieval of type Geometry
```

Comment:
nit: Block comment style here

Reply:
Referred it from here - [https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/293e2e94b1e7[…]260b512ae360675d02bf8d8/test/odbc/mssqlodbc/test_data_types.cpp](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/293e2e94b1e73e0d8260b512ae360675d02bf8d8/test/odbc/mssqlodbc/test_data_types.cpp#L244)